pkg/errors/errors.go

```diff
@@ -364,18 +484,29 @@ func IsIPOutOfRange(err error) bool {
 	return checkError(err, IPOutOfRange)
 }
 
+func IsMultipleErrors(err error) bool {
+	return checkError(err, MultipleErrors)
+}
+
+// checkError checks if the wrappedError has the same MocCode as the err error according to GetMocErrorCode.
+// If the error is not matched by GetMocErrorCode and the error does not have a GRPC code (or is a GRPC Unknown code),
+// it will attempt to match the error strings through string matching (even for multierrors).
 func checkError(wrappedError, err error) bool {
```

Comment:
Can we write some unit tests for this function where the wrappedError not a `MocError`, instead it's an error constructed using errors package from golang with `errors.New()`? Because we cannot always be sure we are passing `MocError` to Is* functions.

Reply:
Yeah I have an existing unit test that covers this "TestCheckErrorBackwardsCompatible/Error_without_explicit_GRPC_Unknown_code_uses_string_matching". Do you think this suffices?